### PR TITLE
Output requirement deleted

### PR DESCRIPTION
**Pull request by ozgurhalilince**
Hi,

We don't want to put generated file to our server. The function already returning buffer image. I think there is no need to output requirement

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -25,7 +22,11 @@ module.exports = async function({
   }
   await page.setContent(html)
   const element = await page.$('body')
-  const buffer = await element.screenshot({ path: output, type, omitBackground: transparent, encoding })
+  const screenShotOptions = { type, omitBackground: transparent };
+  if (output) {
+    Object.assign(screenShotOptions, { path: output });
+  }
+  const buffer = await element.screenshot(screenShotOptions)
```
Comment (frinyvonnick) [inline src/index.js:29]: The previous implementation was good. If output is `undefined` path will be undefined. What do you think of it?

Comment (ozgurhalilince) [reply src/index.js:29]: ah, i understand ur issue. I think assigning initial value fixes it, like `output = ''` . then, we can check if it is not empty @frinyvonnick

Comment (frinyvonnick) [reply src/index.js:29]: I think we don't need to check it. `output` is `undefined` by default so :

```js
const buffer = await element.screenshot({ path: output, type, omitBackground: transparent, encoding })
```

will do the same as : 

```js
const screenShotOptions = { type, omitBackground: transparent };
if (output) {
  Object.assign(screenShotOptions, { path: output });
}
const buffer = await element.screenshot(screenShotOptions)
```

The scope of this PR is to make output optional. Removing this code is enough : 

```js
  if (!output) {	
    throw Error('You must provide an output property.')	
  }
```

**Comment by Heziode** [comment]
Any progress ?

**Comment by frinyvonnick** [comment]
Hi @Heziode, I made the changes myself since a lot of people ask for it and @ozgurhalilince seems to be occupied 👍

**Comment by frinyvonnick** [comment]
I'll publish a version as soon as possible 😄

**Comment by frinyvonnick** [comment]
It has been published in [2.0.0](https://github.com/frinyvonnick/node-html-to-image/releases/tag/v2.0.0) 🚀

**Comment by Heziode** [comment]
@frinyvonnick Thank's :muscle: :fire:

**Comment by weiyunpeng** [comment]
> It has been published in [2.0.0](https://github.com/frinyvonnick/node-html-to-image/releases/tag/v2.0.0) 🚀

When I used yarn to download 2.0.0, I didn't find it.

**Comment by frinyvonnick** [comment]
As you can see on [npm website](https://www.npmjs.com/package/node-html-to-image) the latest version is [2.0.0](https://www.npmjs.com/package/node-html-to-image/v/2.0.0). I tried to install `node-html-to-image` using the following command `yarn add node-html-to-image` it installed version 2.0.0. Maybe this problem comes from a custom config on your computer @weiyunpeng?

**Comment by weiyunpeng** [comment]
---Original---

**Pull request merged**